### PR TITLE
Remove a couple of manual loops in the `PDFViewerApplication.open` method

### PR DESCRIPTION
**Pull request by Snuffleupagus**
Some of the code in this method is *very* old, and we could thus modernize it a little bit by removing a couple of the loops used to build the `getDocument` argument.

### CODE REVIEW

File: web/app.js [hunk 1]
```diff
@@ -919,12 +919,11 @@ const PDFViewerApplication = {
       await this.close();
     }
     // Set the necessary global worker parameters, using the available options.
-    const workerParameters = AppOptions.getAll(OptionKind.WORKER);
-    for (const key in workerParameters) {
-      GlobalWorkerOptions[key] = workerParameters[key];
+    const workerParams = AppOptions.getAll(OptionKind.WORKER);
+    for (const key in workerParams) {
+      GlobalWorkerOptions[key] = workerParams[key];
```
Comment (calixteman) [inline web/app.js:924]: nit: why not using Object.assign ?

**Review by calixteman — APPROVED** [review]
LGTM. Thank you.

File: web/app.js (hunk 1 above)
Comment (Snuffleupagus) [reply web/app.js]: No reason; I've update the patch :-)

**Pull request merged**